app/controllers/orders_controller.rb

```diff
@@ -32,6 +32,7 @@ def new
 
       @marker_options = l1.marker_params
       @map_options = l1.map_params
```

Comment:
This code is duplicated in both `app/controllers/messages_controller.rb` and `app/controllers/orders_controller.rb`. Move logic, to User model.

`current_user.oposite_roles`

and inside the model

`
def oposite_roles
  User.where(role: User.roles[rider? :driver : :rider])
end
`

Therefore in controller

`
 @users = current_user.oposite_roles
`